scripts/PyChop/ISISDisk.py

```diff
@@ -451,5 +451,5 @@ def _removeLowIntensityReps(self, Eis, lines, Ei=None):
             idx1 = (np.abs(Eis - Ei) / np.abs(Eis)) < 0.1
             idx += idx1
         Eis = np.array([Eis[i] for i in range(len(Eis)) if idx[i]])
-        lines = np.array([lines[i] for i in range(len(Eis)) if idx[i]])
+        lines = np.array([lines[i] for i in range(len(lines)) if idx[i]])
```

Comment:
these lines could be simplified even more. `Eis` is a numpy array, so you could just say `Eis=Eis[idx]`. For `lines`, you need to transform it to a numpy array first, so line 454 becomes `lines=np.array(lines)[idx]`